frontend/js/modules/pad-view.js

```diff
@@ -63,6 +64,7 @@ module.exports = (function () {
     c.isAuth    = auth.isAuthenticated();
     c.isGuest   = !c.isAuth;
     c.bookmarks = (c.isAuth ? auth.userInfo().bookmarks.pads : []);
+    c.watchlist = (c.isAuth ? auth.userInfo().watchlist.pads : []);
```

Comment:
Is it possible for userInfo to not have watchlist attribute here as well? (I'm not sure which executes first, so maybe its handled by checking that executes sooner)